scripts/setup-operators.sh

```diff
@@ -104,4 +104,4 @@ done
 echo "Completed OpenShift GitOps operator installation"
 
 echo "Provide cluster-admin access to argocd-application-controller service account"
-oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:openshift-gitops:argocd-cluster-argocd-application-controller
+oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller
```

Comment:
We no longer have to make Argo CD cluster-admin. We can fix it in a different PR

Reply:
> We no longer have to make Argo CD cluster-admin.

This step has been deprecated in day 1 doc too. @ishitasequeira Please remove it

> We can fix it in a different PR

@chetan-rns Do we have jira or github issue ?, if not please create one

Reply:
Will create a new issue @amitkrout